p2p/host/relay/autorelay.go

```diff
@@ -54,6 +54,10 @@ type AutoRelayHost struct {
 	addrs  []ma.Multiaddr
 }
 
+// Warning:  This function should not be used directly by clients activating
+// autorelay. NewAutoRelayHost is used internally by the libp2p.config.NewNode
```

Comment:
this should be `libp2p.New` constructor, the users don't call `libp2p.conig.NewNode` directly either.

Reply:
Corrected